docs/layout.md

```diff
@@ -0,0 +1,899 @@
+        - [ ] Step 4/4. Have fun with Teleport!
+        - [ ] Guides
+    - [ ] Installation
+        - [ ] Linux
+        - [ ] Docker
+        - [ ] Helm
+        - [ ] MacOs
+        - [ ] Windows (tsh client only)
+        - [ ] Installing from source
+        - [ ] Checksums
+        - [ ] Operating System support
+    - [ ] User Manual
+        - [ ] Introduction
+        - [ ] Installing tsh
+        - [ ] User identities
+        - [ ] Logging in
+        - [ ] Explorign the cluster
+        - [ ] Interactive shell
+        - [ ] Copying files
+        - [ ] Sharing sesssions
+        - [ ] Connecting to SSH clusters behind a load balancer
+        - [ ] Web UI
+        - [ ] Using OpenSSH client
+        - [ ] Troubleshooting
+        - [ ] Getting help
+    - [ ] Admin Manual
+        - [ ] Installing
+        - [ ] Definitions
+        - [ ] Teleport Daemon
+        - [ ] Configuration
+        - [ ] Authentication
+        - [ ] Adding and deleting users
+        - [ ] Editing users
+        - [ ] Adding Nodes to the cluster
+        - [ ] Revoking invitations
+        - [ ] Adding a Node located behind NAT
+        - [ ] Labeling Nodes and applications
+        - [ ] Auding Log
+        - [ ] Resources
+        - [ ] Trusted Clusters
+        - [ ] GitHub Oath 2.0
+        - [ ] HTTP CONNECT proxies
+        - [ ] PAM integration
+        - [ ] Using Teleport with OpenSSH
+        - [ ] Certificate rotation
+        - [ ] Ansible integration
+        - [ ] Kubernetes integration
+        - [ ] Storage backeneds
+        - [ ] High Availability
+        - [ ] Upgrading Teleport
+        - [ ] Backing up Teleport
+        - [ ] GitOps
+        - [ ] Migrating backends
+        - [ ] License file
+        - [ ] Troubleshooting
+        - [ ] Getting help
+    - [ ] Production Guide
+        - [ ] Prerequisites
+        - [ ] Designing your cluster
+        - [ ] Firewall configuration
+        - [ ] Installation
+        - [ ] Running Teleport in production
+        - [ ] Security considerations
+    - [ ] FAQ
+        - [ ] Community FAQ
+        - [ ] Whcih version of Teleport is supported
+        - [ ] Commercial Teleport Editions
+        - Section-specific FAQ
+    - [ ] Changelog
+        - Teleport repository README is injected here.
+- [ ] Setup Section
+    - [ ] Guides
+        - w/ Tile landing
+        - [ ] Terraform Provider
+            - [ ] Follow along with our video guide
+            - [ ] Prerequisites
+            - [ ] Step 1/4. Install terraform provider
+            - [ ] Step 2/4. Create a terraform user
+            - [ ] Step 3/4. Create Terraform configuration
+            - [ ] Step 4/4. Apply the configuration
+            - [ ] Next steps
+        - [ ] Local Docker
+            - [ ] Pick your image
+            - [ ] Quickstart using docker-compose
+            - [ ] Quickstart using docker run
+            - [ ] Creating a Teleport user when using Docker quickstart
```

Comment:
Needs updating after https://github.com/gravitational/teleport/pull/7225 gets merged in